<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="3" column="1">
       <widget class="QSlider" name="xSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="srcSizeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Source size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="lensDistSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="einsteinSpinbox"/>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="xSpinbox"/>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="srcSizeSpinbox"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="lensDistSpinbox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X position</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lens distance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="einsteinSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="ySlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Y position</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="ySpinbox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Einstein radius</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Terms</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="markerBox">
        <property name="text">
         <string>Markers</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="srcTypeComboBox">
        <item>
         <property name="text">
          <string>Gauss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Triangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rocket</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="gridBox">
        <property name="text">
         <string>Grid</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Source type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="termsSpinbox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="infTermsCheckbox">
          <property name="text">
           <string>Infinite terms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="actLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="distLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionSave_image_as"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGrid_2">
     <property name="title">
      <string>Grid</string>
     </property>
     <addaction name="actionOff"/>
     <addaction name="action2x2"/>
     <addaction name="action4x4"/>
     <addaction name="action8x8"/>
     <addaction name="action12x12"/>
     <addaction name="actionCustom"/>
    </widget>
    <addaction name="menuGrid_2"/>
    <addaction name="actionMarkers"/>
    <addaction name="actionLegend"/>
    <addaction name="separator"/>
    <addaction name="actionChange_resolution"/>
    <addaction name="separator"/>
    <addaction name="actionDark_mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionMarkers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Markers</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="action2x2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2x2</string>
   </property>
  </action>
  <action name="action4x4">
   <property name="text">
    <string>4x4</string>
   </property>
  </action>
  <action name="action8x8">
   <property name="text">
    <string>8x8</string>
   </property>
  </action>
  <action name="action12x12">
   <property name="text">
    <string>12x12</string>
   </property>
  </action>
  <action name="actionChange_resolution">
   <property name="text">
    <string>Change resolution</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionDark_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
  <action name="actionSave_image_as">
   <property name="text">
    <string>Save image as...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
